CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -349,7 +349,7 @@ CF_EXPORT void * __CFConstantStringClassReferencePtr;
 
 #if DEPLOYMENT_RUNTIME_SWIFT
 
-CF_EXPORT void *__CFConstantStringClassReference[];
+CF_EXPORT void *__CFConstantStringClassReference[] asm("_T010Foundation19_NSCFConstantStringCN");
```

Comment:
I cant currently build foundation on Xcode due to lots of other unrelated errors but given that it seems to be defined in  `CoreFoundation/Base.subproj/SymbolAliases`  and there looks to be a fix in https://github.com/apple/swift-corelibs-foundation/pull/1006 which I assume if for macOS do I just need to make this change for non macOS? something like:

```
#if TARGET_OS_MAC
CF_EXPORT void *__CFConstantStringClassReference[];
#else
CF_EXPORT void *__CFConstantStringClassReference[] asm("_T010Foundation19_NSCFConstantStringCN");
#endif
```